code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -200,6 +201,12 @@
 		xenos += ssd_xeno
 	return xenos
 
+///fetches number of bonus points given to the hive,
+/datum/hive_status/proc/get_evolution_boost()
```

Comment:
just get the lenght, this proc is uneeded

Reply:
I want to add potentially tier two of these or such